### ISSUE TITLE
Tests fails on Linux for Rakudo 2017.07

### ISSUE THREAD

**Issue by nige123** [Bailador/Bailador#228]
Receiving the following test failures for Rakudo/MoarVM 2017.07

```
===> Testing: Bailador:ver('0.0.9')


# Skipping author test. Set AUTHOR_TESTING to enable.
# Skipping author test. Set AUTHOR_TESTING to enable.
# Skipping author test. Set AUTHOR_TESTING to enable.

    # Failed test 'program terminated successfully'
    # at t/20-cli.t line 127
    # expected: '0'
    #      got: '1'

    # Failed test 'text matches 'rx:s/Result\: PASS/''
    # at t/20-cli.t line 130
    #      expected: 'rx:s/Result\: PASS/'
    #      got: ''
    # Looks like you failed 2 tests of 3

# Failed test 'test newly generated app with its test cases'
# at t/20-cli.t line 120

    # Failed test 'text matches 'rx:s/\<h1\>Bailador App\<\/h1\>/''
    # at t/20-cli.t line 189
    #      expected: 'rx:s/\<h1\>Bailador App\<\/h1\>/'
    #      got: ''

    # Failed test 'text matches 'rx:s/Version 0\.0\.1/''
    # at t/20-cli.t line 190
    #      expected: 'rx:s/Version 0\.0\.1/'
    #      got: ''
    # Looks like you failed 2 tests of 2

# Failed test 'test app with watch command without bin directory'
```

**Comment by szabgab** [comment]
(Just editing the report: I put the error message in triple-backticks so they are shown verbatim.)

**Comment by nige123** [comment]
Thanks for the fox - good to know about triple backticks.

**Comment by ufobat** [comment]
I am not sure but to me it seems that #229 is basically the same issue. because 20-cli.t boots up your app, and this is failing.

**Comment by szabgab** [comment]
I think this has been fixed in the new 0.0.10. Please reopen the ticket if the issue persists in 0.0.10 for you.
